Core/GDCore/IDE/CaptureOptions.h

```diff
@@ -0,0 +1,48 @@
+#pragma once
+#include <memory>
+#include <vector>
+
+#include "GDCore/String.h"
+
+namespace gd {
+
+class GD_CORE_API Screenshot {
+ public:
+  Screenshot();
+  virtual ~Screenshot() {};
+
+  void SetTiming(int timing_) { timing = timing_; }
+  const long GetTiming() const { return timing; }
```

Comment:
```suggestion
 long GetTiming() const { return timing; }
```

(returning a primitive is not really needing any const, because you return a copy of it anyway, so the caller can do whatever they want with it, they won't modify the timing inside the class) (i.e: it's not a reference, so you get a copy)